.github/workflows/sdk-featurestore_sample-automation-test-test_featurestore_cli_samples.yml

```diff
@@ -3,7 +3,7 @@
 # Any manual changes to this file may cause incorrect behavior.
 # Any manual changes will be overwritten if the code is regenerated.
 
-name: sdk-featurestore_sample-test_featurestore_cli_samples
+name: sdk-featurestore_sample-automation-test-test_featurestore_cli_samples
```

Comment:
Is that possible to move automation-test out of featurestore_sample folder? For example, move it parallel to featurestore_sample, named as featurestore_sample_automation_test?

Reply:
The test working directory should be inside the featurestore-samples, and I don't see this pattern in other examples.